### PR TITLE
Mg longshot

### PR DESCRIPTION
**Pull request by MG-longshot**
## PR Checklist: (Checked-off items are either done or do not apply to this PR)
 
- [x] I have performed a self-review of my code
- [ ] CHANGELOG.md is updated (if end-user facing)
- [x] Sent notification to software-design channel requesting review

### CODE REVIEW

File: extras/AFC_stepper.py [hunk 1]
```diff
@@ -238,10 +238,15 @@ def prep_callback(self, eventtime, state):
                 if self.load_state == True and self.prep_state == True:
                     self.status = 'Loaded'
                     self.AFC.afc_led(self.AFC.led_ready, led)
-            else:
```
Comment (jimmyjon711) [inline extras/AFC_stepper.py:241]: This else statement need to stay, if a lane is ejected the led stays green

Comment (MG-longshot) [reply extras/AFC_stepper.py:241]: Done

File: extras/AFC_stepper.py [hunk 2]
```diff
@@ -238,10 +238,18 @@ def prep_callback(self, eventtime, state):
                 if self.load_state == True and self.prep_state == True:
                     self.status = 'Loaded'
                     self.AFC.afc_led(self.AFC.led_ready, led)
+            elif self.name == self.AFC.current and self.AFC.STATUS == 'Printing' and self.AFC.lanes[self.unit][self.name]['runout_lane'] != 'NONE':
```
Comment (jimmyjon711) [inline extras/AFC_stepper.py:241]:
```suggestion
            elif self.name == self.AFC.current and self.IDLE.state == 'Printing' and self.AFC.lanes[self.unit][self.name]['runout_lane'] != 'NONE':
```

File: extras/AFC.py [hunk 3]
```diff
@@ -16,7 +15,9 @@ def __init__(self, config):
         self.printer.register_event_handler("klippy:connect",self.handle_connect)
         self.SPOOL = self.printer.load_object(config,'AFC_spool')
         self.ERROR = self.printer.load_object(config,'AFC_error')
+        self.IDLE = self.printer.load_object(config,'idle_timeout')
         self.gcode = self.printer.lookup_object('gcode')
+        self.STATUS= self.IDLE.state
```
Comment (jimmyjon711) [inline extras/AFC.py:20]: Since this will always return the same state and not update, this line can be removed

File: extras/AFC_stepper.py (hunk 2 above)
Comment (MG-longshot) [reply extras/AFC_stepper.py]: changed

**Review by jimmyjon711 — APPROVED** [review]

**Pull request merged**